x-pack/test/functional/services/transform_ui/wizard.ts

```diff
@@ -75,6 +75,47 @@ export function TransformWizardProvider({ getService }: FtrProviderContext) {
       await testSubjects.existOrFail(selector);
     },
 
+    async parseTable(tableSubj: string) {
+      const table = await testSubjects.find(`~${tableSubj}`);
+      const $ = await table.parseDomContent();
+      const rows = [];
+
+      // For each row, get the content of each cell and
+      // add its values as an array to each row.
+      for (const tr of $.findTestSubjects(`~${tableSubj}Row`).toArray()) {
+        rows.push(
+          $(tr)
+            .find('.euiTableCellContent')
+            .toArray()
+            .map(cell =>
+              $(cell)
+                .text()
+                .trim()
+            )
+        );
+      }
+
+      return rows;
+    },
+
+    async assertTableResults(tableSubj: string, column: number, expectedColumnValues: string) {
+      await retry.tryForTime(2000, async () => {
+        // get a 2D array of rows and cell values
+        const rows = await this.parseTable(tableSubj);
+
+        // reduce the rows data to an array of unique values in the specified column
+        const uniqueColumnValues = rows
+          .map(row => row[column])
+          .flat()
+          .filter((v, i, a) => a.indexOf(v) === i);
+
+        uniqueColumnValues.sort();
```

Comment:
I think fetching unique values and sort them can be done shorter with
```
const uniqueColumnValues = [...new Set(rows.map(row => row[column]))].sort();
```
but I guess that's a matter of taste ;-)

Reply:
I tried that before and somehow had issues making it work, leaving as is for now. I usually keep `.sort()` on a separate line without assignment because it mutates the original array.